### PR TITLE
daemon: add a inference.js in model-define plugin for inferencing.

### LINKED ISSUE #331: pipeline: separate the model define plugin into 2 lifecycle
As for the current __model-define__ plugin, we use the one function to handle the different lifecycles: training and serving, this looks a little complicated to plugin developers. Therefore we want to address the issue by the following:

1. add an optional new file "inference.js" in every __model-define__ plugin, which will be called when serving.
2. pass the labelmap to "inference.js".

### PR DESCRIPTION
**Pull request by yorkie**
This simplifies the model-define plugin development, and this fixes #331. The rest work is to update all the builtin model-define plugins:

- [x] `bayesian-classifier`
- [ ] `detectron-fasterrcnn`
- [ ] `detectron-maskrcnn`
- [ ] `fasttext`
- [ ] `tensorflow-cycle-gan`
- [ ] `tensorflow-mobilenet`
- [ ] `tensorflow-resnet`
- [ ] `tfjs-mobilenet`
- [ ] `tfjs-simplecnn`
- [ ] `tfjs-lstm`

### CODE REVIEW

File: packages/plugins/model-define/bayesian-model-define/src/inference.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import { processPredictData, loadModel } from './script';
+
+const boa = require('@pipcook/boa');
+const sys = boa.import('sys');
+
+/**
+ * The inference model.
+ * @param recoverPath The model path to be recovered.
+ */
+export default async function inference (recoverPath: string): Promise<any> {
```
Comment (rickycao-qy) [inline packages/plugins/model-define/bayesian-model-define/src/inference.ts:12]: Shall we give a type definition for inference function in pipcook-core

Comment (yorkie) [reply packages/plugins/model-define/bayesian-model-define/src/inference.ts:12]: Of course :)

**Comment by rickycao-qy** [comment]
@yorkie Could you rebase since there are some conflicting files. And what's the status of this PR. Will we give separate PR for each plugin?

**Comment by yorkie** [comment]
Just closing this because we will have a new design for plugin entry.

**Pull request closed without merging**